STAR-Delegation/CSR-template/template-schema.cddl

```diff
@@ -32,7 +34,7 @@ rsaKeyType = {
   SignatureType: $rsaSignatureType
 }
 
-rsaKeySize = int .ge 2048
+rsaKeySize = uint .ge 2048
```

Comment:
Propose to drop this constraint.

Reply:
OK; this will have a corresponding change in the json-schema template.